src/custom/pages/Claim/styled.ts

```diff
@@ -266,6 +266,9 @@ export const ConfirmOrLoadingWrapper = styled.div<{ activeBG: boolean }>`
   padding: 24px;
   color: white;
   position: relative;
+  display: flex;
```

Comment:
not sure i like this change, can we revert this for now?

Reply:
The flex is just to center the blue spinner which currently for me it looks like this
![Screenshot from 2022-01-06 15-42-31](https://user-images.githubusercontent.com/34926005/148400285-96eb78e8-f280-4220-a58b-2c7b38f9b9de.png)

Reply:
And after it will look like this
![Screenshot from 2022-01-06 15-43-31](https://user-images.githubusercontent.com/34926005/148400407-0d6004c5-e156-4c5e-988b-fe2ab4e79500.png)

Reply:
👍 my fault thought it was the table!